manifests/define.pp

```diff
@@ -1,48 +1,30 @@
 # == Define: fail2ban::define
 #
 define fail2ban::define (
-  $config_file_path         = undef,
-  $config_file_owner        = undef,
-  $config_file_group        = undef,
-  $config_file_mode         = undef,
-  $config_file_source       = undef,
-  $config_file_string       = undef,
-  $config_file_template     = undef,
+  Stdlib::Absolutepath $config_file_path         = "${fail2ban::config_dir_path}/${title}",
+  String $config_file_owner        = $fail2ban::config_file_owner,
```

Comment:
can you align everything at `=` please?

Reply:
I chose not to align at all and use single spaces.